WordPress/src/main/java/org/wordpress/android/ui/stats/StatsUtils.java

```diff
@@ -352,6 +352,18 @@ public static synchronized void logVolleyErrorDetails(final VolleyError volleyEr
         AppLog.e(T.STATS, "Volley Error details: " + volleyError.getMessage(), volleyError);
     }
 
+    public static synchronized  boolean isAuthenticationError(final VolleyError volleyError) {
+        if (volleyError != null && volleyError.networkResponse != null) {
```

Comment:
I think this can be simplified to just:
```
public static synchronized  boolean isAuthenticationError(final VolleyError volleyError) {
    return (volleyError instanceof com.android.volley.AuthFailureError);
}
```